frontend/src/actions/WindowActions.js

```diff
@@ -312,10 +312,10 @@ export function updateDataFieldProperty(property, item, scope) {
   };
 }
 
-export function noConnection(status) {
+export function noConnection({ errorType }) {
```

Comment:
rename the function to `connectionError` to have a name consistent with the action type.